init.c

```diff
@@ -2336,6 +2403,8 @@ static int parse_set (BUFFER *tmp, BUFFER *s, unsigned long data, BUFFER *err)
 	  strfcpy (scratch, tmp->data, sizeof (scratch));
 	  mutt_expand_path (scratch, sizeof (scratch));
 	  *((char **) MuttVars[idx].data) = safe_strdup (scratch);
+      if (mutt_strcmp (MuttVars[idx].option, "debug_file") == 0)
+	    restart_debug ();
```

Comment:
build is broken, certainly because when `DEBUG` is not defined, this line cannot compile.

https://travis-ci.org/neomutt/neomutt/builds/206916932#L483